tests/test_subdomains.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+from math import floor
+
+from conftest import skipif
+from devito import Grid, Function, TimeFunction, Eq, solve, Operator, SubDomainSet
+
+pytestmark = skipif(['yask', 'ops'])
+
+
+class TestSubdomains(object):
```

Comment:
I suggest a test with two  SubDomainSets in the same Operator ?

Reply:
Good idea. Will add this morning.